src/lib.rs

```diff
@@ -1124,18 +1127,18 @@ where
 {
     fn from_iter<T: IntoIterator<Item = (K, V)>>(iter: T) -> Self {
         let mut it = iter.into_iter();
+        let guard = crossbeam::epoch::pin();
```

Comment:
This can use an `unprotected` guard I think

Reply:
Just changed it.